tasks/install.yml

```diff
@@ -47,12 +47,30 @@
 
 - meta: flush_handlers
 
-- name: Install plugins
+- name: Set fluentd dirs ownership
+  file:
+    path: "{{ item }}"
+    owner: td-agent
+    group: td-agent
+    recurse: yes
+    state: directory
+  with_items:
+  - "/etc/td-agent"
```

Comment:
I don't think directories containing configuration files (in `/etc` dir) should be owned by a user which runs a program.